src/Microsoft.DotNet.Interactive.Jupyter/IsCompleteRequestHandler.cs

```diff
@@ -32,11 +34,15 @@ public async Task Handle(JupyterRequestContext context)
     {
         switch (@event)
         {
-            case CompleteCodeSubmissionReceived _:
-                Reply(true, context.JupyterRequestMessageEnvelope, context.JupyterMessageSender);
+            case DiagnosticsProduced diagnosticsProduced:
```

Comment:
Should `CommandFailed` also be handled?

Reply:
No. If `RequestDiagnostics` fails that's a bug.